### PR TITLE
Release trv10 2.0.1

### PR DESCRIPTION
**Pull request by mofahsan**
- remove order.status from pre on_confirm calls
- add cancel_initiated in post on_confirm calls
- add energy_types enums in swagger schema
- remove ev from vehicle variant
- update SOFT-CANCEL && CONFIRM-CANCEL hyphen to underscore.

### CODE REVIEW

File: api/build/build.yaml [hunk 1]
```diff
@@ -2694,6 +2694,14 @@ components:
           type: string
         energy_type:
           type: string
+          enum:
+            - Electric
```
Comment (92shreyansh) [inline api/build/build.yaml]: These all need to follow ENUM format of UPPER_CASE

File: api/components/enum/default/index.yaml [hunk 2]
```diff
@@ -34,9 +34,6 @@ search:
               description: Describes the vehicle category as TWO_WHEELER
               reference: <PR/Issue/Discussion Links md format text
           variant:
-            - code: EV
```
Comment (92shreyansh) [inline api/components/enum/default/index.yaml:37]: Make sure this is also removed from examples

Comment (mofahsan) [reply api/components/enum/default/index.yaml:37]: ev examples updated to AUTO_RICKSHAW

File: api/build/build.yaml (hunk 1 above)
Comment (mofahsan) [reply api/build/build.yaml]: enums updated to upper case

**Pull request merged**